### PR TITLE
Added basic RPC server to GEN

### PR DESCRIPTION
**Pull request by peddybeats**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,9 +1,14 @@
...
+
 all: build
 
 build: protos
 
 protos: .FORCE
+	# Copy proto files for app and fe to be dynamically compilation
 	echo fe app | xargs -n 1 cp -rf protos
+	# Pre-compile genius proto
+	python -m grpc_tools.protoc -I ./protos --python_out=$(GEN_PROTOS) --grpc_python_out=$(GEN_PROTOS) ./protos/genius.proto
```
Comment (PedramPejman) [inline Makefile]: nit: line too long

File: gen/src/client.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+import protos.genius_pb2_grpc as genius_grpc
+
+def main():
+    """Example client that calls the Handshake.Shake RPC method"""
+
+    channel = grpc.insecure_channel('localhost:35000')
+    stub = genius_grpc.HandshakeStub(channel)
+    response = stub.Shake(genius.HandshakeRequest(name='Pedram'))
+    print("Server responded with: " + response.message)
+
```
Comment (peddybeats) [inline gen/src/client.py:15]: del

File: Makefile [hunk 3]
```diff
@@ -1,9 +1,14 @@
+GEN_PROTOS=./gen/src/protos
+
 all: build
 
 build: protos
 
 protos: .FORCE
+	# Copy proto files for app and fe to be dynamically compilation
 	echo fe app | xargs -n 1 cp -rf protos
+	# Pre-compile genius proto
```
Comment (peddybeats) [inline Makefile]: nit: too long

**Review by peddybeats — COMMENTED** [review]
LGTM

**Pull request merged**